src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -106,7 +106,10 @@ handshake11 a b = handshake a b <> initialise a True <> initialise b True
 
 -- TODO: Restrict the initial state so that a=b=1 is not allowed.
 me :: a -> a -> CircuitConcept a
-me a b = fall a ~> rise b <> fall b ~> rise a
+me a b = fall a ~> rise b <> fall b ~> rise a <> invariantConcept [rise a, rise b]
```

Comment:
I may keep invariant concept in `Basic` as this matches the names of the other standard concept types. I will implement a `never` function though for this purpose.